src/Admin/EventAdmin.php

```diff
@@ -197,8 +197,14 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                         return;
                     }
 
-                    $qb->andWhere(sprintf('%s.postAddress.postalCode', $alias).' LIKE :postalCode');
-                    $qb->setParameter('postalCode', $value['value'].'%');
+                    $value = array_map('trim', explode(',', strtolower($value['value'])));
+                    $idExpression = $qb->expr()->orX();
```

Comment:
Why `idExpression` ?

Reply:
like our lord and savior @HeahDude once said: "copy pasted with the ass."